client/commonFramework/add-loop-protect.js

```diff
@@ -5,7 +5,10 @@ window.common = (function(global) {
   } = global;
 
   loopProtect.hit = function hit(line) {
-    var err = `Error: Exiting potential infinite loop at line ${line}.`;
+    var err = `Error: Exiting potential infinite loop at line ${line}.
+    To disable loop protection: add "// noprotect" to your code.
```

Comment:
Removing double quotes fixed `ReferenceError: loopProtect is not defined`

![image](https://cloud.githubusercontent.com/assets/1938039/12071691/f90cd22e-b0de-11e5-9713-1c101ea2466f.png)

Following was my local change on top of this PR commit:
```
diff --git a/client/commonFramework/add-loop-protect.js b/client/commonFramework/add-loop-protect.js
index 098b61e..b227d43 100644
--- a/client/commonFramework/add-loop-protect.js
+++ b/client/commonFramework/add-loop-protect.js
@@ -5,10 +5,10 @@ window.common = (function(global) {
   } = global;
 
   loopProtect.hit = function hit(line) {
-    var err = `Error: Exiting potential infinite loop at line ${line}.
-    To disable loop protection: add "// noprotect" to your code.
-    Beware that if you do have an infinite loop in your code
-    this will crash your browser.`;
+    var err = "Error: Exiting potential infinite loop at line ${line}."
+    + "To disable loop protection: Write \\/\\/noprotect as first line."
+    + "Beware that if you do have an infinite loop in your code"
+    + "this will crash your browser.";
     console.error(err);
   };
 
diff --git a/client/commonFramework/update-preview.js b/client/commonFramework/update-preview.js
index 4567f67..be34660 100644
--- a/client/commonFramework/update-preview.js
+++ b/client/commonFramework/update-preview.js
@@ -15,7 +15,7 @@ window.common = (function(global) {
     window.__err = new Error(
       "Potential infinite loop at line " +
       line +
-      ". To disable loop protection: add \"// noprotect\" to your code." +
+      ". To disable loop protection: Write \\/\\/noprotect as first line. " +
       "Beware that if you do have an infinite loop in your code" +
       " this will crash your browser."
     );
```